### DISCUSSION TITLE
Collapse / expand all sections button in sidebar

### DISCUSSION THREAD

**Discussion by mfridman** [facebook/docusaurus#5323] [Ideas]
In large nested sidebars, when you want to understand (or ctrl+f) find something it'd be useful to expand/collapse ALL sections.

This button could be placed anywhere in the sidebar (usually towards the top)

![CleanShot 2021-08-09 at 19 55 21@2x](https://user-images.githubusercontent.com/6278244/128788636-2d874a76-1a45-451c-b1f7-f255b23c13f9.png)

E.g., 

Click to collapse _everything_
<img width="147" alt="CleanShot 2021-08-09 at 19 50 52@2x" src="https://user-images.githubusercontent.com/6278244/128788470-32f3cf8e-21d7-4561-ad63-2235d9a4fd91.png">

Click to expand _everything_ (including all nested sections)
<img width="225" alt="CleanShot 2021-08-09 at 19 50 48@2x" src="https://user-images.githubusercontent.com/6278244/128788478-61a19e66-bad6-4755-94a3-74272bb4b16c.png">

**Comment by slorber** [comment]
Why not, but I suggest to add this to Canny otherwise we'll likely forget about this discussion entry

**Reply by mfridman** [reply]
Created a Canny entry. 

https://docusaurus.io/feedback/p/collapse-expand-all-sections-button-in-sidebar

Should we close this discussion?

**Reply by slorber** [reply]
Thanks

There's not any "closed" status in discussion and I wouldn't find it anyway if I were looking for bugs or feature requests to work on, but I marked it as "accepted"
